src/messages.js

```diff
@@ -0,0 +1,9 @@
+var req = require.context('../lang', true, /\.json.*$/);  
+var exports = {};
+
+req.keys().forEach(function (file) {
```

Comment:
`keys()` will not work at all in Internet Explorer. I'm not positive babel will transpile stuff like that.